manager/test/api/src/test/resources/test-plans/searching-testPlan.xml

```diff
@@ -1,46 +1,69 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Nit: it's hard to see what's changed here because the indentation has changed. Would it be possible to change that back so it's a little clearer in the diff?

Reply:
BTW, you may need to look at the full diff to see what I mean. It doesn't show much on the PR page.